3. Installation/4. Manual Installation/Running in a sub folder.md

```diff
@@ -1,14 +1,47 @@
 # Running in a sub folder
 
-If you want to run your server in a sub folder like `https://your.domain.com/chat/` you should read this guide.
+If you want to run your server in a sub folder like `https://your.domain.com/chat/` you have to setup a reverse proxy.
+See [Run Rocket.Chat behind a SSL Reverse Proxy](/3.%20Installation%2F4.%20Manual%20Installation%2FConfiguring%20SSL%20Reverse%20Proxy.md)
+
+But in case of using apache, you have to tweak the configuration a bit!
+First you have to put the Rewrite/Proxy related parts into a Location-tag.
+Lets say, you want to provide the service under the above mentioned subfolder "chat", this is what this part of the config should look like:
+
+```
+        <Location /chat>
+                Order allow,deny
+                Allow from all
+
+                RewriteEngine On
+                RewriteCond %{HTTP:Upgrade} =websocket [NC]
+                RewriteRule /var/www/(.*)           ws://localhost:3000/$1 [P,L]
+                RewriteCond %{HTTP:Upgrade} !=websocket [NC]
+                RewriteRule /var/www/(.*)           http://localhost:3000/$1 [P,L]
+                
+                ProxyPassReverse http://localhost:3000
+        </Location>
+```
+If you compare it to the lines provided in the above linked reverse proxy documentation, you may notice a difference in the RewriteRule - there is an additional `/var/www`.
+That must be changed to your DocumentRoot (if it's not already your DocumentRoot).
+If you pass the whole path, including the DocumentRoot to the rocket.chat server, you'll get a 404 response saying
+```
+Unknown path
+```
+because you configured `/chat` as subfolder (actually it's parsed out of the ROOT_URL you provided) but the rocket.chat server gets a request for `/var/www/chat`, detects a missmatch and returns the above mentioned error.
+
+**Additional notes:**
+- Using the RewriteBase Directive won't work
+- The ProxyPassReverse can be used inside a Location-tag and the first parameter must be omitted in this case
+- You don't need to configure a Location for the root folder `/` when using rocket.chat in a subfolder (its likely configured for other stuff already in this case)
+- For those of you beeing curious about details: the above mentioned path-handling is in webapp.js ...just search for "Unknown path"
+
 
 ## Running the application
-To run in a sub folder you should pass the evironment variable `ROOT_URL` to the process.
+To run in a sub folder you should pass the environment variable `ROOT_URL` to the process.
 For example:
 
 ```shell
 ROOT_URL=https://your.domain.com/chat/ node main.js
 ```
 
-## Configuring the application
-Set the `Site URL` under `Admin > General` to your **domain only** like `https://your.domain.com/`
+If you followed the manual installation guide, you may already have configured (and persisted) this evironment variable - in this case you can ignore this advice.
```

Comment:
You got the previous instance of this but can you change:

`evironment variable` -> `environment variable`